tests/nlu/base/test_components.py

```diff
@@ -44,7 +45,15 @@ def test_all_arguments_can_be_satisfied(component_class):
     }
 
     for req in component_class.requires:
-        assert req in provided_properties, "No component provides required property."
+        if isinstance(req, Tuple):
+            for r in req:
+                assert (
+                    r in provided_properties
+                ), "No component provides required property."
+        else:
+            assert (
+                req in provided_properties
+            ), "No component provides required property."
```

Comment:
maybe it makes sense to make the logic a separate function in the code so that we can test it without re-doing the logic here